product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -38,18 +40,26 @@ release.
     Adding or removing a pair doesn't need a restart of Postgres or even a
     reload of the configuration.
 
--   Group Commit can't be combined with [CAMO](durability/camo/) or [Eager All-Node
-    Replication](consistency/eager/). Eager Replication currently works only by using the
-    global PGD commit scope.
+-   Group Commit can't be combined with [CAMO](durability/camo/) or [Eager
+    All-Node Replication](consistency/eager/). Eager Replication currently works
+    only by using the global PGD commit scope.
 
--   Transactions using Eager Replication can't yet execute DDL. The TRUNCATE command is allowed.
+-   Transactions using Eager Replication can't yet execute DDL. The TRUNCATE
+    command is allowed.
 
--   Parallel apply isn't currently supported in combination with Group Commit. Make sure to disable it when using Group Commit by either:
-    - Setting `num_writers` to 1 for the node group using [`bdr.alter_node_group_config`](/pgd/latest/reference/nodes-management-interfaces/#bdralter_node_group_config).
-    - Using the GUC `bdr.writers_per_subscription`. See [Configuration of generic replication](/pgd/latest/reference/pgd-settings/#generic-replication).
+-   There currently is no protection against altering or removing a commit
```

Comment:
@djw-m we need to bring this back and revert all changes with respect to GC-PA since we still do not support PA with GC.

Reply:
Done